managed-ledger/src/main/java/org/apache/bookkeeper/mledger/LedgerOffloaderMXBean.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.bookkeeper.mledger;
+
+import org.apache.bookkeeper.common.annotation.InterfaceAudience;
+import org.apache.bookkeeper.common.annotation.InterfaceStability;
+import org.apache.bookkeeper.mledger.util.StatsBuckets;
+
+
+/**
+ * Management Bean for a {@link LedgerOffloader}.
+ */
+@InterfaceAudience.LimitedPrivate
+@InterfaceStability.Stable
+public interface LedgerOffloaderMXBean {
+
+    /**
+     * The ledger offloader name.
+     *
+     * @return ledger offloader name.
+     */
+    String getDriverName();
+
+    /**
+     * Record the offload total time.
+     *
+     * @return offload time per topic.
+     */
+    long getOffloadTime(String topic);
+
+    /**
+     * Record the offload error count.
+     *
+     * @return offload errors per topic.
+     */
+    long getOffloadErrors(String topic);
+
+    /**
+     * Record the offload rate to storage.
+     *
+     * @return offload rate per topic.
+     */
+    long getOffloadBytes(String topic);
```

Comment:
OffloadRate is calculated in `LedgerOffloaderMetrics`

Reply:
Do we record the rate or the data bytes? It is confusing.

Reply:
this metrics actually is a rate, calculate when collect metrics(bytes/interval)

Reply:
> Do we record the rate or the data bytes? It is confusing.

this metrics actually is a rate, calculate when collect metrics(bytes/interval)

Reply:
I see. Because we calculate the rate when collecting metrics, we should keep this name and description consistent.  Should we change the description to `Record the offloaded data in bytes` or similar things?

Reply:
> I see. Because we calculate the rate when collecting metrics, we should keep this name and description consistent. Should we change the description to `Record the offloaded data in bytes` or similar things?

you are right, I'll change it